source/server/server.cc

```diff
@@ -379,6 +379,20 @@ void InstanceImpl::initialize(const Options& options,
     dispatcher_->initializeStats(stats_store_, "server.");
   }
 
+  // The broad order of initialization from this point on is the following:
+  // 1. Statically provisioned configuration (bootstrap) are loaded.
+  // 2. Cluster manager is created and all primary (i.e. with endpoint assignments not provisioned
+  //    through xDS) clusters are initialized.
```

Comment:
nit: can you clarify this includes DNS as it might not be obvious. Same elsewhere if there are any other applicable comments.

Reply:
To make sure I understand. I thought DNS clusters are primary. Am I wrong?

Reply:
Yes that's correct. I just meant that people might think that primary is only static clusters and not DNS. It might be worth clarifying that primary also includes DNS, but nbd.

Reply:
Ok, I have clarified comments.